.circleci/config.yml

```diff
@@ -48,6 +48,9 @@ jobs:
             cd ../untagged-ids
             ./run.sh ../../migration-utils/build/index.js
             ./run.sh ../../codegen/build/index.js
+            cd ../camel-cased
```

Comment:
I'm pretty good with a new test directory/db; fwiw just talking out loud, I wonder if we could add a table or two to the existing `schema-misc` that had a few camel cased names?

Reply:
Yep, that's what I was thinking. Granted, we don't have a great written-down heuristic for "when / when not to create a new testing directory"...kind of figuring it out as we go. :-)